tests/haven/test-read_sas.R

```diff
@@ -0,0 +1,100 @@
+# Write relevent tests for the function in here
+# Consider the type of function:
+#   - is it deterministic or statistic?
+#   - is it worth checking for errors/warnings under particular conditions?
+
+
+data_sas <- read_sas("./skinproduct_attributes_seg.sas7bdat")
+data_csv <- read.csv("./skinproduct_attributes_seg.csv", check.names = F)
+
+
+#####################################
+#####		compare content		#####
+#####################################
+
+#compare column 1
+test_that("Column 1 matches"
+	, expect_equal(
```

Comment:
i think i would be inclined to do something like 
`expect_true(all(data_csv$ProductKey == data_sas$ProductKey))`